### ISSUE TITLE
Issuer States Update

### ISSUE THREAD

**Issue by andywalton** [Neufund/platform-frontend#2879]
**TL;DR**
We need to allow our Issuers to list their ETO campaigns earlier, currently our process requires them to have their Investment Terms, Term Sheet & Nominee all setup before we can list them. We want to change this so that we can list them early and say Coming soon and show their public company page with only marketing information completed. This epic helps us achieve this. 

Read the below for more detail:

**Background**
Our Issuer process needs to change to the following:

0. **Step 0:** Registration & KYC

1. **Step 1:** User puts in all marketing data in - state for the issuer **“Preview”**

2. **Step 2:** User submits for listing via Dashboard _**<- this step is NEW!!!**_
- [x] ETO approved and listing in the public state **“Coming soon”** for investor with Founder’s quote on the ETO card & widget
- [ ] BUT all investment term details on the listing page are labeled “TBA”
- [ ] The term sheet tag is gray & inactive in the widget
- [ ] All legal agreements on the listing page are hidden

3. **Step 3:** ETO configuration (User puts in all investment data in and uploads term sheet)

4. **Step 4:** User submits for fundraising
- [ ] ETO approved and listing public state **“Campaign”**  for investor with Founder’s quote on the ETO card & widget  
- [ ] BUT all Investment Terms are shown on the listing page with details
- [ ] The term sheet tag is green & active 

5. **Step 5:** User enables whitelisting - public state for investor **“Whitelist”**

6. **Step 6:** **Presale** countdown

7. **Step 7:** **Public sale** countdown

... the rest then continues as usual.

**Tasks**
So we need to edit our Issuer and Investor views to allow for the above change.

Tickets associated with this Epic are related to the Multi ETO products epic (see here #2547) and they are improvements to our current setup.

States we need to care about - we ask our Issuers for different information based on these states:
![image](https://user-images.githubusercontent.com/609657/57372225-54e6fa00-7195-11e9-93e8-aecba0fae340.png)

Also, we will need to factor in two "submit" for approval call to action buttons:

1. As an Issuer, after I add my company and marketing information I will be asked to submit my application for review inorder to continue to...
2. After completing my Investment terms etc. I will be asked to submit inorder to list these investment terms. 

---

Please ask Aga, Andy or Marcin questions related to any tickets associated with this "mini" epic.
